src/components/Greeting/shared/ButtonUi.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Button } from "@mui/material";
+
+type Props = {
+  children: string;
+};
+
+const style = {
+  color: "secondary.contarstText",
+  boxShadow: "none",
+  "&:hover": {
+    boxShadow: "none",
+    color: "primary.contrastText",
+  },
+};
+
+export const ButtonUi: React.FC<Props> = (props: Props) => {
```

Comment:
```suggestion
export const ButtonUi: React.FC<Props> = ({ children }: Props) => {
```

Reply:
move to shared and rename to SecaondaryButton

Reply:
doen

Reply:
done